src/App.js

```diff
@@ -1,23 +1,38 @@
-import logo from './logo.svg';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Outlet } from 'react-router-dom';
 import './App.css';
+import { fetchUser } from './features/user/userSlice';
+import { fetchReservatins } from './features/reservations/reservationSlice';
+import SideBar from './components/SideBar';
+import LoginPage from './components/LoginPage';
 
 function App() {
```

Comment:
- [ ] I made a few more reservations.  The "reservations" section still has some dates scrambled:


![image](https://user-images.githubusercontent.com/57421823/215004276-a4fbfec8-46c6-48b8-8d76-d3b7a6ba5da3.png)

Kindly fix this bug, so your users know the reservation dates.  Also, you have a typo here:

![image](https://user-images.githubusercontent.com/57421823/215004347-b97e3c4f-e345-4347-a1d5-b4a2f4d6f856.png)